circleci/check-make-generate

```diff
@@ -10,6 +10,14 @@ set -e
 
 USAGE="Usage: check-make-generate"
 
+# Files to ignore when checking for diffs
+# TODO: Consider allowing consumers to specify additional files
```

Comment:
This seems like a nice-to-have. You could probably do it pretty simply either using a comma/semicolon/whatever-separated list or one of these guys https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html

Reply:
Ended up tackling this! And tested thoroughly.

```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate --files-to-ignore
Usage: check-make-generate --files-to-ignore file1,file2,...,fileN
```
```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate --wrong-param-name a,b,c
Usage: check-make-generate --files-to-ignore file1,file2,...,fileN
```
```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate --files-to-ignore a,b,c --extra-param
Usage: check-make-generate --files-to-ignore file1,file2,...,fileN
```
```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate
Running 'make generate' and checking for diffs...

Found diffs involving the following files:

.npmrc_docker
.npmrc

Please run 'make generate'.
```
```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate --files-to-ignore .npmrc_docker
Running 'make generate' and checking for diffs...

Found diffs involving the following files:

.npmrc

Please run 'make generate'.
```
```
arsalan@macbook:family-portal$ ../ci-scripts/circleci/check-make-generate --files-to-ignore .npmrc_docker,.npmrc
Running 'make generate' and checking for diffs...
Found no diffs. 'make generate' has been run.
```